src/handlers/guilds/GUILD_UPDATE.ts

```diff
@@ -30,6 +30,8 @@ export async function handleGuildUpdate(data: DiscordGatewayPayload) {
         if (Array.isArray(cachedValue) && Array.isArray(value)) {
           const different = (cachedValue.length !== value.length) ||
             cachedValue.find((val) => !value.includes(val)) ||
+            // TODO: check if this really works hehe
+            // @ts-ignore this works ts is wrong
```

Comment:
"this works ts is wrong," please refrain from adding these trivial comments, and add meaningful and descriptive reason as to why `// @ts-ignore` is added